src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Think about which variables should be local in the method and which should be class-level. Since 'colorSupplier' is used only in 'getRandomBall' method, it could be a local variable inside the method unless there is a specific reason to keep it at the class level.